### PR TITLE
PS-8525 Add the 5.6.51-93.0 release notes (5.6)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/conf.py
	new file:   source/release-notes/Percona-Server-5.6.51-93.0.rst
	modified:   source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.51-93.0.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.6.51-93.0
+includes all the features and bug fixes available in
+`MySQL 5.6.51 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-51.html>`_
+in addition to enterprise-grade features developed by Percona. 
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7856`: A server exit when auto-incremented column was updated ina partitioned table.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-5.6.51-93.0.rst]: updated ina -> in a

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.51-93.0.rst]: fixed

File: doc/source/release-notes/Percona-Server-5.6.51-93.0.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.6.51-93.0
+includes all the features and bug fixes available in
+`MySQL 5.6.51 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-51.html>`_
+in addition to enterprise-grade features developed by Percona. 
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7856`: A server exit when auto-incremented column was updated in a partitioned table.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-5.6.51-93.0.rst]: A server exit -> A server exited
when auto-incremented column -> when an auto-incremented column

**Review by kaushikpuneet07 — APPROVED** [review]
LGTM

File: doc/source/release-notes/Percona-Server-5.6.51-93.0.rst (hunk 2 above)
Comment (kaushikpuneet07) [reply doc/source/release-notes/Percona-Server-5.6.51-93.0.rst]: is this review comment fixed ? if not please fix it .

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**